testFif.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import argparse
+import shutil
+import os
+import FiF
+from FiF import Folder
+
+#parser = argparse.ArgumentParser()
+#args = parser.parse_args()
+
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
Let's use more informative class name, smth like
```suggestion
class Md5TestCase(unittest.TestCase):